docs/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'imaplib2'
+copyright = '2021, Piers Lauder'
```

Comment:
I think we should use `copyright = '2021 The imaplib2 contributors'` because all contributors still own the copyright of the lines of the code they contribute.

Reply:
> I think we should use `copyright = '2021 The imaplib2 contributors'` because all contributors still own the copyright of the lines of the code they contribute.

:+1: see 938d89f